src/Xamarin.Android.Build.Tasks/Microsoft.NET.Workload.Android/WorkloadManifest.in.json

```diff
@@ -7,7 +7,12 @@
         "Microsoft.Android.Sdk",
```

Comment:
I just noticed, the name is not correct. It should be `sdk-manifest.json`, right?

Reply:
Maybe that `sdk-manifest.json` is in a spec somewhere, but renaming the file seems to break:

```
> mv .\bin\dotnet\sdk-manifests\6.0.100\microsoft.net.workload.android\WorkloadManifest.json .\bin\dotnet\sdk-manifests\6.0.100\microsoft.net.workload.android\sdk-manifest.json
> .\bin\dotnet\dotnet.exe build .\samples\HelloMaui\HelloMaui.Android\HelloMaui.Android.csproj
...
error NETSDK1139: The target platform identifier android was not recognized.
```

I think `WorkloadManifest.json` is what is actually implemented.

This repo might be helpful if someone needs an example of a workload: https://github.com/jonathanpeppers/maui-workload

Reply:
I see. On that note are you building the package as DotnetPlatform type ?

Reply:
It looks like we need to put `PackageType=DotnetPlatform` or `PackageType=MSBuildSdk` in our own packages. I'll check on that.

Reply:
Fixed the ones in xamarin-android here: https://github.com/xamarin/xamarin-android/pull/5636